src/utils/tableDataUtils.ts

```diff
@@ -0,0 +1,99 @@
+import { getOffset } from '@/helpers'
+
+interface TableDataParams {
+  getSingleEntity: (mesh: string, query: string | null, params: { size: number; offset: string | null }) => any
+  getAllEntities: (params: { size: number; offset: string | null }) => any
+  getAllEntitiesFromMesh: (mesh: string) => any
+  mesh: string
+  size: number
+  query: string | null
+  offset: string | null
+}
+
+function sortEntities(
+  items: {
+    name: string
+    mesh: string
+  }[],
+) {
+  return [...items].sort((a, b) => (a.name > b.name ? 1 : a.name === b.name ? (a.mesh > b.mesh ? 1 : -1) : -1))
+}
+
+const getItems = (response: {
+  total: number
+  items: {
+    name: string
+    mesh: string
+  }[]
+}) => {
+  const r = response
+
+  if (!r.total) {
+    return [r]
```

Comment:
If the total is 0 we return whole response as a single-element Array and lower we return [...items]

Reply:
Probably
`return [...r.items]`